src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -39,6 +39,11 @@ export class PiaIntakeService {
     createPiaIntakeDto: CreatePiaIntakeDto,
     user: KeycloakUser,
   ): Promise<GetPiaIntakeRO> {
+    // update submittedAt column if it is first time submit
+    createPiaIntakeDto.submittedAt = this.updatedSubmittedAt(
```

Comment:
this will make `createPiaIntakeDto.submittedAt` empty for the subsequent calls as it returns null otherwise.

You may need to pass the whole dto to the function, and update if needed